tests/integration_tests/test_applications.py

```diff
@@ -29,7 +29,7 @@
             "-t",
             "LST-1",
             "--model_version",
-            "prod5",
+            "prod4",
```

Comment:
Moving back to prod4?

Reply:
No, I reverted the wrong commit apparently. I committed other changes to a different branch that I didn't want to include in this PR and got confused with the commit strings. I am still not good enough with git commands and stuff.

I will fix this, make sure all the tests pass and then merge this.